app/presenters/base_presenter.rb

```diff
@@ -0,0 +1,18 @@
+class BasePresenter < SimpleDelegator
+  def self.map(collection, view)
+    collection.map { |element| new(element, view) }
+  end
+
+  attr_reader :object, :view
+
+  def initialize(object, view)
+    @object = object
+    @view = view
+
+    super(@object)
+  end
+
+  def h
```

Comment:
What is `h` meant to stand for?

Reply:
hahaha the documentation PR explained the uses of this method. But anyway, it is a helper method for use view/helpers methods inside the presenter. This is kind of thing that looks like the I18n.t instead typing I18n.translate every time. I didn't use "v" because h appears better to think that a presenter is a kind a middle layer between helpers/views and models.

Reply:
oh okay